src/OpenLoco/src/Ui/Windows/MusicSelection.cpp

```diff
@@ -131,6 +140,15 @@ namespace OpenLoco::Ui::Windows::MusicSelection
 
         setSortMode(*window, Jukebox::MusicSortMode::original);
 
+        // Set status bar text
+        const auto& config = Config::get().audio;
+        uint16_t numEnabled = 0;
+        for (uint16_t i = 0; i < Jukebox::kNumMusicTracks; i++)
+        {
+            numEnabled += config.customJukebox[i];
+        }
```

Comment:
I think this could be replaced with `numEnabled = std::ranges::count(Config::get().audio, true)`